utils/RQLParser.ts

```diff
@@ -0,0 +1,117 @@
+export enum QueryTypes {
+  FILTER = "FILTER",
+  SORT = "SORT",
+}
+
+export enum Operators {
+  INCLUDE = "INCLUDE",
+  EXCLUDE = "EXCLUDE",
+}
+
+interface Queries {
+  operator: Operators;
+  key: string;
+  value: string;
+  type: QueryTypes;
+}
+
+interface FilterQueryValue {
+  operator: Operators;
+  value: string;
+}
```

Comment:
I think these types can be reused. Can we keep them in separate files?

Reply:
Done.